hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -207,6 +216,33 @@ public static void createSavepointFile(String basePath, String commitTime, Confi
     return copy;
   }
 
+  public List<HoodieRecord> generateInsertsWithHoodieAvroPayload(String commitTime, int limit) throws
```

Comment:
rest of the class works with a trip payload?  is it too much work to just replace that with HoodieAvroPayload fully?

Reply:
The HoodieTestDataGenerator is used across the project, there may be tests relying on the TripsPayload schema to do some validations hence introduced a new one.